### PR TITLE
Project Review  

### PR DESCRIPTION
**Pull request by RaphDasilva**
-Am going to do the following.
--Review my project.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Linters
+
```
Comment (RaphDasilva) [inline .github/workflows/linters.yml:2]: Please fix linters error...To make your project standard.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,357 @@
...
+<body>
+    <div class="nav">
+        <div class="logo"><img src="media/YouTube-Logo.wine.svg" alt="logo" width="100px"></div>
+        <div class="search-placeholder">
+           <div class="search">
+            <input type="text" placeholder="Search.." name="search">
+            <button type="submit" class="button-submit"><i class="fa-sharp fa-solid fa-magnifying-glass"></i></button>
+            </div>
+            <span><i class="fa-sharp fa-solid fa-microphone"></i></span>
+        </div>
```
Comment (RaphDasilva) [inline index.html:20]: Please use nav tag for this section...these are good for SEO and can make your code more readable and understandable.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,357 @@
...
+
+        <div class="head-icons">
+
+            <div><i class="fa-solid fa-square-plus"></i></div>
+            <div><i class="fa-sharp fa-solid fa-bell"></i></div>
+            <div><i class="fa-solid fa-user"></i></div>
+        </div>
+    </div>
```
Comment (RaphDasilva) [inline index.html:28]: Please use ul tag instead, these are good for SEO and can make your code more readable and understandable.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,357 @@
...
+              <div class="icons">
+                <div class="like-dislike">
+                    <button type="submit" class="like"><i class="fa-solid fa-thumbs-up"></i><span>488</span></button>
+                    <button type="submit" class="dislike"><i class="fa-solid fa-thumbs-down"></i></button>
+                </div>
+                <button type="submit" class="share"><i class="fa-solid fa-share"></i><span>Share</span></button>
+                <button type="submit" class="clip"><i class="fa-solid fa-scissors"></i><span>Clip</span></button>
+                <button type="submit" class="save"><i class="fa-solid fa-floppy-disk"></i><span>Save</span></button>
+                <button type="submit" class="more"><i class="fa-solid fa-ellipsis"></i></button>
+            </div>
```
Comment (RaphDasilva) [inline index.html:60]: Please Use recommended HTML semantic tags...It will help you site to rank well in the search engine .

**Review by RaphDasilva — COMMENTED** [review]
Hi @RaphDasilva ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @RaphDasilva    👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**